packages/office-ui-fabric-react/src/components/ProgressIndicator/__snapshots__/ProgressIndicator.test.tsx.snap

```diff
@@ -22,11 +22,7 @@ exports[`ProgressIndicator renders ProgressIndicator correctly 1`] = `
       aria-valuetext={undefined}
       className="ms-ProgressIndicator-progressBar"
       role="progressbar"
-      style={
```

Comment:
Maybe add another test to ProgressIndicator.test.tsx so we have both determinate and indeterminate

Reply:
@lynamemi, good call. I've added a test so that there is one with a % complete and one without.